### PR TITLE
Temporal: Fix name of constructor

### PR DESCRIPTION
**Pull request by ptomato**
I mistakenly committed something slightly different in my code review suggestion for #4306 than what I had modified locally.

### CODE REVIEW

**Review by ctcpip — APPROVED** [review]
you need a 262 maintainer to review, but LGTM

**Review by Ms2ger — APPROVED** [review]

**Pull request merged**
